source/_includes/installation/operating_system.md

```diff
@@ -71,46 +77,119 @@ To boot Home Assistant OS, the BIOS needs to have UEFI boot mode enabled and Sec
 </div>
 
 Typically an internal medium like S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC is used for the x86-64 boot medium. Alternatively, an external medium can be used such as a USB SDD, though this is not recommended.
 
 To write the HAOS image to the boot medium on your x86-64 hardware, there are 2 different methods:
 
-1. Write the HAOS disk image from your desktop computer directly to the boot medium (e.g. using a USB to S-ATA adapter).
-If you can use this method, proceed to "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)" and follow all steps. If you have non-removable internal mediums or don't have the necessary adapter, try the next method instead.
+  **Method 1**: Recommended method. It also works on laptops and PCs with internal hard disks. This method boots Ubuntu from a USB device and installs the Home Assistant Operating System from there.
 
-2. Create a "live operating system" on a USB device running e.g. Ubuntu ([how-to guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Insert it into your system and boot the live operating system. Then follow from step 2 in "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)".
+  **Method 2**: With this method, you write the Home Assistant Operating disk image directly onto a boot medium. The steps are a bit more complex. If you have non-removable internal mediums (for example because you are using a laptop) or do not have the necessary adapter (for example an USB to eMMC adapter) use method 1 instead.
 
-{% details "Ubuntu dependencies for Etcher" %}
+### Method 1: Installing HAOS via Ubuntu from a USB device
 
-When installing Etcher on Ubuntu you may need to install the fuse dependency first,
-to do this run the following commands in the terminal:
+#### Required material
 
-```bash
-sudo add-apt-repository universe
-sudo apt update
-sudo apt install libfuse2
-```
+- Computer
+- The target x86-64 hardware, on which you want to install the Home Assistant Operating System (HAOS)
+- USB device
+- Internet connection
 
-{% enddetails %}
+#### To install HAOS via Ubuntu from a USB device
+
+1. **Notice**: This procedure will write the Home Assistant Operating System onto your device.
+   - This means you will lose all the data as well as the previously installed operating system.
+   - Back up your data before continuing with the next step.
+2. Create a so called *live operating system* on a USB device:
+   - Follow the ([Ubuntu instructions](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)) on writing an Ubuntu iso file onto a USB device.
+3. Insert the USB device into the system on which you want to run Home Assistant.
+   - Boot the live operating system.
+   - The system then starts Ubuntu.
+4. In Ubuntu, open a browser and download the image to your computer.
+      - Copy the URL for the image.
```

Comment:
To be able to copy the URL the person need to navigate to this site. At that point we can just make it an actual link instead.